draft-ietf-pquip-pqc-engineers.md

```diff
@@ -279,22 +303,27 @@ Finally, in their evaluation criteria for PQC, NIST is assessing the security le
 
 For example, to provide some context, one would need 20 million noisy qubits to break RSA-2048 in 8 hours {{RSAShor}}{{RSA8HRS}} or 4099 stable (or logical) qubits to break it in 10 seconds {{RSA10SC}}.
 
-For structured data such as public-key and signatures, instead, CRQCs can fully solve the underlying hard problems used in classic cryptography (see Shor's Algorithm). Because an increase of the size of the key-pair would not provide a secure solution short of RSA keys that are many gigabytes in size {{PQRSA}}, a complete replacement of the algorithm is needed. Therefore, post-quantum public-key cryptography must rely on problems that are different from the ones used in classic public-key cryptography (i.e., the integer factorization problem, the finite-field discrete logarithm problem, and the elliptic-curve discrete logarithm problem).
+For structured data such as public-keys and signatures, instead, CRQCs can fully solve the underlying hard problems used in classic cryptography (see Shor's Algorithm). Because an increase of the size of the key-pair would not provide a secure solution short of RSA keys that are many gigabytes in size {{PQRSA}}, a complete replacement of the algorithm is needed. Therefore, post-quantum public-key cryptography must rely on problems that are different from the ones used in classic public-key cryptography (i.e., the integer factorization problem, the finite-field discrete logarithm problem, and the elliptic-curve discrete logarithm problem).
 
+## Quantum side-channel attacks
```

Comment:
I'm not sure discussing this is worthwhile, given what a can of worms it is.  But I could be convinced otherwise.

Reply:
I also prefer not to discuss about it.

Reply:
I do think it is an important point to mention; engineers who build crypto chips, or who integrate tightly with crypto chips should at least be aware that there's a can of worms here.

I'll adjust this PR to tone this down and say something like "Be aware this is an open research area". See if you like that better.

Reply:
wfm

Reply:
Looks good to me.